src/sagemaker_inference/environment.py

```diff
@@ -43,7 +43,7 @@ class Environment(object):
     Attributes:
         module_name (str): The name of the user-provided module. Default is inference.py.
         model_server_timeout (int): Timeout in seconds for the model server. Default is 60.
-        model_server_workers (int): Number of worker processes the model server will use.
+        model_server_workers (str): Number of worker processes the model server will use.
```

Comment:
The doc string has a mismatch with the actual method. The method returns a string instead of an int: https://github.com/aws/sagemaker-inference-toolkit/blob/master/src/sagemaker_inference/environment.py#L95

Reply:
Ah, that's weird..
Is a reasonable solution to have the method return an int, instead? Similarly to model_server_timeout which parses the string env variable to an int? It feels like a miss during initial dev which we should rectify.
I can't think of a reasonable backwards-compatibility issue, but let me know if you think there is one.

Reply:
The only issue I can think of is that mms property `default_workers_per_model` is defaulted to the number of available CPUs/GPUs. Therefore, we are not forcing a value on the env var `SAGEMAKER_MODEL_SERVER_WORKERS` (which pass the value to self.model_server_workers). 

In the case of SAGEMAKER_MODEL_SERVER_WORKERS is not injected, we cannot do `int(self.model_server_workers)` because it will be NoneType.

Reply:
Gotcha. Thanks for the clarification =)